src/app/src/pages/eval/components/store.ts

```diff
@@ -100,7 +100,9 @@ function extractUniqueStrategyIds(strategies?: Array<string | { id: string }> |
   const strategyIds =
     strategies?.map((strategy) => (typeof strategy === 'string' ? strategy : strategy.id)) ?? [];
 
-  return Array.from(new Set([...strategyIds, 'basic']));
+  // Filter out 'multilingual' - it's now a pure config mechanism that lifts languages to global config
+  // Tests generated with language modifiers appear in 'basic' (Cross-Language Attack) section
+  return Array.from(new Set([...strategyIds, 'basic'])).filter((id) => id !== 'multilingual');
```

Comment:
Does this prevent downstream errors? if not we should revert changes to this file.